packages/wonder-blocks-typography/src/__docs__/typography.stories.js

```diff
@@ -0,0 +1,294 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import type {StoryComponentType} from "@storybook/react";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {OptionItem, SingleSelect} from "@khanacademy/wonder-blocks-dropdown";
+import {
+    Title,
+    HeadingLarge,
+    HeadingMedium,
+    HeadingSmall,
+    HeadingXSmall,
+    BodySerifBlock,
+    BodySerif,
+    BodyMonospace,
+    Body,
+    LabelLarge,
+    LabelMedium,
+    LabelSmall,
+    LabelXSmall,
+    Tagline,
+    Caption,
+    Footnote,
+} from "@khanacademy/wonder-blocks-typography";
+
+import ComponentInfo from "../../../../.storybook/components/component-info.js";
+import {name, version} from "../../package.json";
+import TypographyArgTypes from "./typography.argtypes.js";
+
+const typographyDescription = `Typography. \`wonder-blocks-typography\`
+provides a set of standardized components for displaying text in a consistent
+way. This includes components for headings, paragraphs, and text
+labels.\n\n\n ### Usage
+
+\`\`\`jsx
+import {Body, Title} from "@khanacademy/wonder-blocks-typography";
+
+<Title>Title: Hello, world!</Title>
+<Body>This is just a regular paragraph</Body>
+\`\`\`
+`;
+
+export default {
+    title: "Typography",
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {component: typographyDescription},
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    argTypes: TypographyArgTypes,
+};
+
+export const ControlProps: StoryComponentType = (args) => <Title {...args} />;
```

Comment:
I didn't name this `Default` because I thought that would be confusing considering there's no default in the Typography package. Please let me know if you can think of something better than `ControlProps` for this.